main.js

```diff
@@ -64,7 +64,9 @@ var obstacleSpike = {
     ctx.closePath();
     ctx.fillStyle = "black";
     ctx.fill();
-  }
+  },
+  xmin: 30,
+  ymin: 100
```

Comment:
Yes, that's the idea. I think most shapes are pretty squae-ish so that will work. Even for the ciricle stuff (saw) it's probably not noticeable... (compared to actual collision). First order approximation :-P

Reply:
:+1: